static/js/stream_popover.js

```diff
@@ -367,6 +372,15 @@ exports.register_click_handlers = function () {
         build_starred_messages_popover,
     );
 
+    $("body").on("click", ".move-topic-dropdown .list_item", (e) => {
+        e.preventDefault();
+
+        const stream_name = stream_data.maybe_get_stream_name(
+            Number.parseInt(stream_widget.value(), 10),
+        );
+        ui_util.decorate_stream_bar(stream_name, stream_header_colorblock, false);
+    });
```

Comment:
Can you add a comment on why this doesn't use `stopPropagation` list most of our click handlers?